pyqtgraph/graphicsItems/ROI.py

```diff
@@ -1200,14 +1200,16 @@ class Handle(UIGraphicsItem):
     sigClicked = QtCore.Signal(object, object)   # self, event
     sigRemoveRequested = QtCore.Signal(object)   # self
     
-    def __init__(self, radius, typ=None, pen=(200, 200, 220), parent=None, deletable=False):
+    def __init__(self, radius, typ=None, pen=fn.mkPen(200, 200, 220), parent=None, deletable=False,
+                 activePen = fn.mkPen(255, 255,0)):
```

Comment:
Should generally avoid using mutable objects in a function signature (see for example http://docs.python-guide.org/en/latest/writing/gotchas/).